### PR TITLE
login and signup

### PR DESCRIPTION
**Pull request by anasm11**

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+body{
+    background-color: #eeeeee;
```
Comment (Mohitk391) [inline login.css]: Suggestion: use css variables instead of color codes

File: signup.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+body{
+    background-color: #eeeeee;
```
Comment (Mohitk391) [inline signup.css]: Use Css variable

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
+body{
+    background-color: #eee;
```
Comment (Mohitk391) [inline styles.css]: Use Css Variable

**Review by Mohitk391 — APPROVED** [review]
Try to implement suggestions for better code practices, overall code looks clean and consistent...UI looks nice as well..you can merge this branch

File: signup.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+.form-header{
+    color:var(--primary-color);
+    margin:0;
+}
+
+.form-body{
+    display:flex;
+    flex-direction:column;
+    gap:0.5rem;
```
Comment (abhisheksingh98) [inline signup.css]: You can use px for less than 10px

File: signup.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    height:90vh;
+    display:flex;
+    align-items:center;
+    justify-content:center;
+}
+
+.form{
+    background-color: white;
+    border-radius: 1rem;
+    box-shadow:0 0 0.2rem 0rem;
```
Comment (abhisheksingh98) [inline signup.css]: You can use px for less than 10px instead of rem

**Review by JuvitaSaini09 — APPROVED** [review]

File: signup.css (hunk 2 above)
Comment (anasm11) [reply signup.css]: done. thanks for review.

**Pull request merged**